test/codeu/chat/server/DeleteConversationTest.java

```diff
@@ -84,4 +85,36 @@ public void testdeleteMultipleConversations() {
 	  assertTrue(model.conversationById().first(testConversation1.id) == null&& model.conversationById().first(testConversation2.id)==null);
```

Comment:
Please break this line up. It is too long:
assertTrue(model.conversationById().first(testConversation1.id) == null 
  && model.conversationById().first(testConversation2.id)==null);